tests/integration/components/modals/edit-user-modal-test.js

```diff
@@ -0,0 +1,14 @@
+import { module, test } from 'qunit';
+import { setupIntegrationTest } from 'open-event-frontend/tests/helpers/setup-integration-test';
+import hbs from 'htmlbars-inline-precompile';
+import { render } from '@ember/test-helpers';
+
+module('Integration | Component | modals/edit-user-modal', function(hooks) {
+  setupIntegrationTest(hooks);
+
+  test('it renders', async function(assert) {
+    this.set('isOpen', false);
+    await render(hbs`{{modals/edit-user-modal isOpen=isOpen}}`);
+    assert.ok(this.element.innerHTML.trim().includes('Edit user system roles'));
```

Comment:
This should be `assert.ok(this.element.innerHTML.trim().includes('Update System Roles'));`